### PR TITLE
fix: hide deploy aad manifest command when aad is not enabled

### PR DESCRIPTION
**Pull request by SLdragon**
Currently, even sso feature is not enabed, the command "deploy aad manifest" still exist in command palette.

E2E TEST: Will add test plan for this

### CODE REVIEW

File: packages/vscode-extension/src/extension.ts [hunk 1]
```diff
@@ -56,6 +56,7 @@ import {
 import { loadLocalizedStrings } from "./utils/localizeUtils";
 import { ExtensionSurvey } from "./utils/survey";
 import { ExtensionUpgrade } from "./utils/upgrade";
+import { isAADEnabled } from "@microsoft/teamsfx-core";
```
Comment (tecton) [inline packages/vscode-extension/src/extension.ts:59]: This import can be merged with Line 19.

File: packages/vscode-extension/src/extension.ts [hunk 2]
```diff
@@ -623,6 +624,17 @@ async function initializeContextKey() {
     workspaceUri && (await isM365Project(workspaceUri.fsPath))
   );
 
+  const workspacePath = workspaceUri?.fsPath;
+  if (workspacePath) {
```
Comment (tecton) [inline packages/vscode-extension/src/extension.ts:628]: Is this intended to check whether it's teamsfx project? If so, you could use `isTeamsFxProject`.

File: packages/vscode-extension/src/extension.ts (hunk 1 above)
Comment (SLdragon) [reply packages/vscode-extension/src/extension.ts:59]: Thanks, will update

File: packages/vscode-extension/src/extension.ts (hunk 2 above)
Comment (SLdragon) [reply packages/vscode-extension/src/extension.ts:628]: Yes, you're right, will updatge

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**